src/app/beer_garden/api/http/base_handler.py

```diff
@@ -210,7 +211,7 @@ def write_error(self, status_code, **kwargs):
         )
 
         self.set_header("Content-Type", "application/json; charset=UTF-8")
-        self.set_status(code)
+        self.set_status(code, reason=message)
```

Comment:
This allows clients, such as tornado's `http_client` (used in unit tests), access to the `message` value. Otherwise, the `http_client` writes its own `reason` based on the status code.